xonsh/completer.py

```diff
@@ -317,13 +333,15 @@ def module_complete(self, prefix):
 
     def _quote_paths(self, paths, start, end):
         out = set()
+        quotable = ' (){}-,+*'
```

Comment:
Yeah, you're right.  I want to keep parens since I think they lead to syntax errors, but if dashes don't have that problem, then that's fine.  I'll go with ``" `\t\r\n${}%*()"`` for now (including backtick as well since that is used for globbing).  Maybe we should also only quote `%` if we're on Windows, since it isn't used for variable expansion elsewhere.